src/components/Callout/index.js

```diff
@@ -90,7 +90,7 @@ const Callout = ({
       css({
         backgroundColor: colorScheme.getCSSColor('overlay'),
         [mUp]: {
-          boxShadow: colorScheme.getCSSColor('overlayShadow')
+          boxShadow: `inset 0px -50px 50px -30px colorScheme.getCSSColor('overlayShadow')`
```

Comment:
Alignment is off:
![Screenshot 2021-01-27 at 14 44 40](https://user-images.githubusercontent.com/20746301/105999649-3a159100-60ae-11eb-9886-c6f0f7c67501.jpg)